packages/@react-spectrum/listbox/test/ListBox.test.js

```diff
@@ -58,7 +61,6 @@ describe('ListBox', function () {
     offsetWidth = jest.spyOn(window.HTMLElement.prototype, 'clientWidth', 'get').mockImplementation(() => 1000);
     offsetHeight = jest.spyOn(window.HTMLElement.prototype, 'clientHeight', 'get').mockImplementation(() => 1000);
     scrollHeight = jest.spyOn(window.HTMLElement.prototype, 'scrollHeight', 'get').mockImplementation(() => 48);
-    jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => cb());
```

Comment:
legacy timers added support for raf that conflicted with the old mock we had for raf's

we could leave this as legacy by moving our mock for raf to be after useFakeTimers

however, it seems better to update the test to be a bit more accurate to the real life usecase

Reply:
This can be verified in the reverse by changing
```
jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => cb());
```
to
```
jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => setTimeout(cb, 0));
```